### ISSUE TITLE
`Set.prototype.difference` is missing

### ISSUE THREAD

**Issue by zloirock** [oven-sh/bun#2309]
https://bun.sh/blog/bun-v0.5.7#new-methods-on-set

![image](https://user-images.githubusercontent.com/2213682/222935379-2c7eef24-c3e3-41b2-b0da-3e453b0b6f62.png)

```js
console.log(Set.prototype.difference);   // => undefined
console.log(Set.prototype.intersection); // => [Function: intersection]
```

Most likely it's a WebKit issue, but this proposal is disabled by default even in the latest Safari TP and announced in your blog, so it's an issue also for you -)

**Comment by Jarred-Sumner** [comment]
will be added after next webkit upgrade (I didn't realize it was missing)

https://github.com/WebKit/WebKit/pull/10783

**Comment by Electroid** [comment]
Fixed in Bun 0.5.8

**Comment by zloirock** [comment]
It seems not:
![image](https://user-images.githubusercontent.com/2213682/226523485-38a32660-eb98-4885-bf7f-b5b547fdee3c.png)

**Comment by zloirock** [comment]
Fixed in 0.6.0.
